x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts

```diff
@@ -92,6 +97,16 @@ interface GetAlertParams {
   index?: string;
 }
 
+interface GetAlertSummaryParams {
+  id?: string;
+  gte: string;
+  lte: string;
+  featureIds?: string | string[];
+  filter?: object;
```

Comment:
Is this [type](https://github.com/elastic/kibana/blob/main/packages/kbn-es-query/src/es_query/types.ts#L74) correct for this object?

```
export interface BoolQuery {
  must: estypes.QueryDslQueryContainer[];
  must_not: estypes.QueryDslQueryContainer[];
  filter: estypes.QueryDslQueryContainer[];
  should: estypes.QueryDslQueryContainer[];
}
```

Reply:
For the `filter` attribute, it should probably be `estypes.QueryDslQueryContainer[]`

Reply:
yes, it will work, I won't be able to do it with io-ts, but I can have it in the alert client with some casting.